src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/LegacyPbaAccountServiceImpl.java

```diff
@@ -47,36 +46,31 @@
         return pbaNumbers;
     }
 
-    private  List<String> getPbaNumbersFromPaymentAccount(List<PaymentAccount> paymentAccountsEntity) {
+    private List<String> getPbaNumbersFromPaymentAccount(List<PaymentAccount> paymentAccountsEntity) {
 
         List<String> paymentAccountPbaNumbers = new ArrayList<>();
 
-        if (!paymentAccountsEntity.isEmpty()) {
-
-            paymentAccountsEntity.forEach(paymentAccount ->
-
+        paymentAccountsEntity.forEach(paymentAccount ->
                 paymentAccountPbaNumbers.add(paymentAccount.getPbaNumber().trim())
+        );
 
-            );
-        }
         return paymentAccountPbaNumbers;
     }
 
     private List<String> getPbaNumbersFromPaymentAccount(List<PaymentAccount> userMapPaymentAccount, List<PaymentAccount> paymentAccountsEntity) {
 
         List<String> paymentAccountPbaNumbers = new ArrayList<>();
 
-        if (!paymentAccountsEntity.isEmpty()) {

```

Comment:
This line was removed as it is redundant - this check is already taking place in line 31, therefore the mutated condition of this line was untestable/impossible to cover